TCAT/Controllers/HomeOptionsCardViewController.swift

```diff
@@ -225,7 +229,8 @@ class HomeOptionsCardViewController: UIViewController {
             self.infoButton.alpha = 1
 
             self.searchBar.snp.remakeConstraints { (make) in
-                make.leading.top.equalToSuperview()
+                make.leading.equalToSuperview()
+                make.top.equalToSuperview().inset(self.searchBarTopOffset)
```

Comment:
can we move the layout constants from being a variable of the class to constants at the beginning of the function? that way you won't need to do `self.searchBarTopOffset`

Reply:
It's necessary for computations in other functions